src/web/app/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE web-app PUBLIC "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN" "http://java.sun.com/dtd/web-app_2_3.dtd">
-<web-app>
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+                             http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+  version="3.1"
```

Comment:
Did you check what Tomcat version is needed to support this? Does it need to be documented?
Seems it would work with Tomcat 8, but not anymore with Tomcat 7.
@randomorder you have some pulse on the deploys, is Tomcat 7 still common, or used at all?

Reply:
Tomcat 7 is still supported by the Apache crew receiving regular patch releases, it has an announced EOL of March 31 2021: https://tomcat.apache.org/tomcat-70-eol.html dropping it now may be a tad early.

We still see some tomcat 7 deployments at client sites, though few.

Reply:
Hi @aaime  @mprins

We've been using Tomcat 8 for all new installations for a while now and upgraded tomcat 7 to 8 for all geoserver upgrades done for our clients because we had some issues with updating / upgrading GeoServer on tomcat 7 based deployments (bytecode processing libraries if I remember correctly).

Summarizing, it is not a big deal in my opinion because:

For fresh installations
Tomcat 9 is default from Ubuntu 20.04 LTS repos
Tomcat 8 and 9 from Ubuntu 18.04 LTS repos
Tomcat 8 and 9 on CentOS 7 (haven't looked at CentOS 8 but is safe to assume that it will be 8+)

For GeoServer updates / upgrades it is not unlikely that they'll have to update tomcat 7 anyways due to the bytecode processing library bug